mono-api/mono-api.js

```diff
@@ -0,0 +1,74 @@
+'use strict'; 
+
+const http = require('https');
+const BASE_URL = 'api.monobank.ua';
+/*
+    this token should be passed from repository layer, it's here just for tests
+*/ 
+const TOKEN = 'uMLmvCFm6kA4N8mrflxayXMcZPqFUrxpm9q_CxBsMkaY';
+
+function createRequestBody(path, token) {
+    return {
+        hostname: BASE_URL,
+        path: path,
+        method: 'GET',
+        headers: {
+            'X-Token': token,
+            'Content-Type': 'application/json; charset=UTF-8',
+        }
+    };
+}
+
+function getData(path, token) {
+    const options = createRequestBody(path, token);
+    const req = http.request(options);
+    req.on('response', res => {
+        if (res.statusCode !== 200) {
+            networkErrorHandler(res);
+        } else {
+            res.on('data', body => {
+                try {
+                    const data = JSON.parse(body);
+                    console.log(data);
+                    return data;
+                } catch(e) {
+                    console.log(`Error: something went wront while trying to parse data: ${e}`);
+                }
+            });
+        }
+    });
+
+    req.on('error', err => {
+        try {
+            console.log(err.toString());
+            const e = JSON.parse(err);
+            console.log(`Error: ${e.errorDescription}`);
+            return e;
+        } catch(e) {
+            console.log(`Error: something went wront while trying to parse err: ${e}`);
+        }
+    }); 
```

Comment:
Maybe, move it to function like `networkErrorHandler`?

Reply:
I thought that `networkErrorHandler` would work just with network error, not with mono-API errors
I don't think that `networkErrorHandler` should know about API errors

Reply:
Create another function, like `apiErrorHandler`?

Reply:
we talked about it with @Hud-Volodymyr, he said that's repository module responsibility to handle API errors
and mono API has just one field in error-model, that's `errorDescription`, so I don't know is it worth it or not